src/modules/auth/wrappers/with-auth/with-auth.tsx

```diff
@@ -61,7 +55,7 @@ export const withAuth = (
 
 		return (
 			<>
-				{!isLoggedIn && <Loading fullscreen />}
+				{!tosAccepted && <Loading fullscreen />}
 				<WrappedComponent {...props} />
 			</>
 		);
```

Comment:
preferably you never render the wrappedComponent when the toc is not accepted yet.
Otherwise the wrapped component starts initializing and throwing errors since it does not have a valid session yet:

http://localhost:3200/vrt
![image](https://user-images.githubusercontent.com/1710840/165476844-176805a9-7cb3-4f58-910e-2341e5685194.png)

```
if (!tosAccepted) {
	return <Loading fullscreen />
}
return <WrappedComponent {...props} />
```